runtime/cudaq/qis/qubit_qis.h

```diff
@@ -19,7 +19,7 @@
 #include <cstring>
 #include <functional>
 
-#define __qpu__ __attribute__((annotate("quantum")))
+#define __qpu__ __attribute__((annotate(cudaq::kernelAnnotation)))
```

Comment:
This is defined in 'Frontend' directory, hence build error `error: no member named 'kernelAnnotation' in namespace 'cudaq'`.
Need to refactor the constant strings in a common file. See discussion https://github.com/NVIDIA/cuda-quantum/pull/1794#discussion_r1672470040

Reply:
ok. got it. I will do that.